### DISCUSSION TITLE
news/20231107/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#270] [Website Discussions]
# news/20231107/

Today was day two of the Final Cut Pro Creative Summit.

https://fcp.cafe/news/20231107/

**Comment by benblaser60** [comment]
"you can't just copy and paste Events between Library bundles"
I don't understand this, because we actually can copy Events between Libraries. I must not understand the sentence.

"Apple DOES care." Apple is singular.

**Reply by latenitefilms** [reply]
I’m referring to moving Events between Libraries at the Finder level. Yes, you can move Events between Libraries within the Final Cut Pro interface, but so far no one’s ever been able to do this outside of Final Cut Pro, aside from FCPXML which is incredibly limited.

**Reply by benblaser60** [reply]
Ah, that makes sense, got it.
